File: VB/GenerateUserFriendlyId.Module/Module.vb
Imports DevExpress.ExpressApp
Imports GenerateUserFriendlyId.Module.BusinessObjects
Imports GenerateUserFriendlyId.Module

Namespace GenerateUserFriendlyId.Module
	Public NotInheritable Partial Class GenerateUserFriendlyIdModule
		Inherits ModuleBase

		Public Sub New()
			InitializeComponent()
		End Sub
		Protected Overrides Function GetDeclaredExportedTypes() As System.Collections.Generic.IEnumerable(Of System.Type)
			Return New System.Type() { GetType(Sequence) }
		End Function
	End Class
End Namespace

File: CS/XPO/GenerateUserFriendlyId.Module/Module.cs
using DevExpress.ExpressApp;
using GenerateUserFriendlyId.Module.BusinessObjects;
using GenerateUserFriendlyId.Module;

namespace GenerateUserFriendlyId.Module {
	public sealed partial class GenerateUserFriendlyIdModule : ModuleBase {
		public GenerateUserFriendlyIdModule() {
			InitializeComponent();
		}
		protected override System.Collections.Generic.IEnumerable<System.Type> GetDeclaredExportedTypes() {
			return new System.Type[] { typeof(Sequence) };
		}
	}
}